### DISCUSSION TITLE
求助，一直提示无法加载社区插件

### DISCUSSION THREAD

**Discussion by abcvav** [obsidianzh/forum#94] [Q&A常见问题] [answered]
如图所示，一直提示无法加载社区插件，现在是0.12.10版本，重装没用，挂vpn没用，换电脑没用，有人知道怎么解决嘛？？
![image](https://user-images.githubusercontent.com/47599849/125598931-345ddc48-3b61-4b60-b4ba-f858b1603056.png)

**Comment by jxjxjhjh** [comment]
需要科学上网

**Comment by makerjackie** [comment, marked as answer]
重复问题，答案可见: https://github.com/obsidianzh/forum/discussions/25

**Comment by JImmyQM** [comment]
买个梯子

**Comment by abcvav** [comment]
我挂梯子一点用都没有，还是这样，你们都有用吗

**Reply by abcvav** [reply]
现在又可以了。。。。。。。。。

**Comment by JImmyQM** [comment]
梯子不稳定吧

abcvav ***@***.***>于2021年7月28日 周三下午1:24写道：
